source/_docs/style-guide.md

```diff
@@ -469,6 +469,35 @@ Use panels for extraneous but useful information such as troubleshooting section
 ```
 </div>
 
+## Tables
+
+Markdown tables are defined like this:
```

Comment:
The goal of this entry should be to add a standard for our practice of noting feature access based on service level, so we need more context here. 

Can you revise this intro to describe our new standard including where on the page to place a feature access table (towards the top)?